pallets/subtensor/src/lib.rs

```diff
@@ -854,6 +861,7 @@ pub mod pallet {
         PowRegistrationAllowed(u16, bool), // --- Event created when POW registration is allowed/disallowed for a subnet.
         TempoSet(u16, u16),             // --- Event created when setting tempo on a network
         RAORecycledForRegistrationSet(u16, u64), // Event created when setting the RAO recycled for registration.
+        WeightsMinStake(u64),
```

Comment:
comment on this one would be nice. Also doc comments instead of line comments would be nice around here in general so people can see them when they hover stuff in their editor, though I forget if doc comments actually work in this position in substrate's outer macro pattern 🤔